tdclient/client.py

```diff
@@ -194,6 +194,19 @@ def tail(self, db_name, table_name, count, to=None, _from=None, block=None):
         """
         return self.api.tail(db_name, table_name, count, to, _from, block)
 
+    def change_database(self, db_name, table_name, new_db_name):
+        """Change the table's database into another database.
```

Comment:
For me, it was unclear what "changing database" means until I try the functionality on the real TD databases. Thus, I personally believe the docstring can be more articulate like: *Move a target table from its original database to a new destination database.*

Method name should be as is because it's same as the API spec though.

Reply:
Thank you for your feedback! 75a5840